proxy/proxy.go

```diff
@@ -162,14 +162,20 @@ type PassiveHealthCheck struct {
 
 	// The maximum probability of unhealthy endpoint to be dropped out from load balancing for every specific request
 	MaxDropProbability float64
+
+	// The maximum ratio of unhealthy endpoints in the list of all endpoints PHC will check in case of all endpoints
```

Comment:
Proper go doc starts with the var name: `MaxUnhealthyEndpointsRatio is the maximum ratio of unhealthy ...`